python/extractSerpent2GCs.py

```diff
@@ -29,8 +33,13 @@ def makePropertiesDir(
         os.mkdir(outdir)
 
     # the constants moltres looks for:
-    goodStuff = ['BETA_EFF', 'Chit', 'Chid', 'lambda', 'Diffcoef', 'Kappa',
+    goodStuff = ['BETA_EFF', 'lambda', 'Diffcoef', 'Kappa',
                  'Sp0', 'Nsf', 'Invv', 'Remxs', 'Fiss', 'Nubar', 'Flx']
+    if use_chit:
```

Comment:
Hi @smpark7 It seems that requiring the user to specify this could be avoided, particularly since the moltres input file itself declares whether or not delayed neutron precursors will be used. 

If chit is available, it should be recorded in goodstuff. Same with chid and chip. If any of them are not available, perhaps they could be stored as None. Then, they are all available in the input data regardless of the choice made in the moltres input file in by the moltres user at runtime.

Reply:
I've changed this so that this script extracts all three CHI data, which are always available in a Serpent 2 output file. Moltres will now decide whether to use CHIP or CHIT in ```CoupledFissionKernel``` or ```CoupledFissionEigenKernel``` based on the existing ```account_delayed``` flag.